.github/workflows/debian-executable-release.yml

```diff
@@ -19,6 +19,21 @@ jobs:
         run: |
          git fetch --prune --unshallow
          echo "::set-output name=TAG_NAME::$(git describe --tags $(git rev-list --tags --max-count=1))"
+      - name: process version and release
+        if: contains(steps.get-tag.outputs.TAG_NAME, 'rc' )
+        id: process-release-version
+        run: |
+          releaseVersionString=${{ steps.get-tag.outputs.TAG_NAME}}
+          echo "::set-output name=version::$(echo "$releaseVersionString" | cut -d- -f1)"
+      - name: Setting file path
+        id: set-file-path
+        run: |
+          if [[ ${{ steps.get-tag.outputs.TAG_NAME }} == *"rc"* ]]; then
+            file_path=dist/deb/${{ github.event.inputs.formula }}_${{ steps.process-release-version.outputs.VERSION }}.${{ steps.get-sha.outputs.SHA_SHORT }}-1_amd64.deb
+          else
+            file_path=dist/deb/${{ github.event.inputs.formula }}_${{ steps.get-tag.outputs.TAG_NAME }}.${{ steps.get-sha.outputs.SHA_SHORT }}-1_amd64.deb
+          fi
+          echo "FILE_PATH=$file_path" >> $GITHUB_OUTPUT
```

Comment:
Artifact is also uploaded to github, Please do test that as well.

Reply:
Verified upload of github asset to the particular release